plugins/http/server_intercepter.go

```diff
@@ -59,6 +59,9 @@ func (h *ServerInterceptor) AfterInvoke(invocation operator.Invocation, result .
 	span := invocation.GetContext().(tracing.Span)
 	if wrapped, ok := invocation.Args()[0].(*writerWrapper); ok {
 		span.Tag(tracing.TagStatusCode, fmt.Sprintf("%d", wrapped.statusCode))
+		if wrapped.statusCode >= 400 {
+			span.Error(string(wrapped.body))
```

Comment:
You should only tag error codes rather than the whole body. It doesn't make sense.

Reply:
done